h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -409,3 +409,56 @@ def _metrics_class(model_json):
         else:
             raise NotImplementedError(model_type)
         return [metrics_class, model_class]
+
+    def convert_H2OXGBoostParams_2_XGBoostParams(self, num_class):
+        '''
+        Given a H2OXGBoost model, this method will generate the corresponding parameters that should be used
+        by native XGBoost in order to give exactly the same result assuming that the same dataset (dervied from
+        h2oFrame) is used to train the native XGBoost model.
+
+        Follow the following steps to compare H2OXGBoost and native XGBoost:
+
+        1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction
+        h2oModelD = H2OXGBoostEstimator(**h2oParamsD) # parameters specified as a dict()
+        h2oModelD.train(x=myX, y=y, training_frame=trainFile) # train with H2OFrame trainFile
+        h2oPredict = h2oPredictD = h2oModelD.predict(trainFile)
+
+        2. derive the DMatrix from H2OFrame
+        nativeDMatrix = h2oModelD.convert_H2OXGBoostParams_2_XGBoostParams()
+
+        3. derive the parameters for native XGBoost:
+        nativeParams = trainFile.convert_H2OFrame_2_DMatrix(myX, y, h2oModelD)
+
+        4. train your native XGBoost model and generate a prediction
```

Comment:
Uppercase first word in steps 2-4 and end with colon or period. (A colon or period is required in Step 1 also.)
2. Derive the DMatrix from H2OFrame.
nativeDMatrix = h2oModelD.convert_H2OXGBoostParams_2_XGBoostParams()
3. Derive the parameters for native XGBoost:
nativeParams = trainFile.convert_H2OFrame_2_DMatrix(myX, y, h2oModelD)
4. Train your native XGBoost model and generate a prediction.